repo/packages/C/chronos/0/marathon.json

```diff
@@ -28,6 +28,8 @@
       "portIndex": 0,
       "protocol": "HTTP"
     }
-  ]
-
+  ],
+  "labels": {
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{chronos/mesos-framework-name}}"
```

Comment:
Let me talk to you in person, @jsancio. Chronos is like Cassandra in that it generates the framework name from other information (a config variable called mesos framework name).